### DISCUSSION TITLE
[29 marzo 2023] Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#244] [Esami]
Usando gli assiomi dell’algebra di Boole,

- verificare la seguente identità:

<div align="center">

![image](https://github.com/user-attachments/assets/58d83021-5cfe-483d-8c69-de508afb3b16)

</div>

- scrivere la forma POS normale
  scrivere la forma POS canonica

**Comment by Dieal** [comment]
![IMG_20250108_110713_861](https://github.com/user-attachments/assets/cd11f42d-0c6b-49cf-b46d-5e6babf66569)
![IMG_20250108_110715_784](https://github.com/user-attachments/assets/3d2d4cb2-af8b-4ab2-9b42-666e6eeb30f9)

Le canoniche si sarebbero potute ottenere direttamente dall'equazione, visto che:
- SOP: A'XX + XB'C
- POS: (A' + B' + X) (A' + X + C)
dove "X" è don't care
